### PR TITLE
Create 'Sym' class and setup basic stuff for HW 1

### PR DESCRIPTION
**Pull request by Surya-06**
Summary:
1. Update read me file to indicate current status / projected plan.
2. Update main.py to display a placeholder string
3. Create 'Sym' class with required methods for mode and entropy

### CODE REVIEW

File: Code/main.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env python3
 
-__author__ = "SE Team Project 11 - HW 0"
+__author__ = "SE Team Project 11 - HW 1"
 __version__ = "0.1.0"
 __license__ = "MIT"
 
-
+# BEGIN - Older test functions - keeping them for CI sanity
```
Comment (kaushikjadhav01) [inline Code/main.py:7]: Pls remove older functions. We have a release for this. And HW1 was just about creating a good repo for which we will have new HW2 tests now

Comment (kaushikjadhav01) [inline Code/main.py:7]: Pls remove older functions. We have a release for this. And HW1 was just about creating a good repo for which we will have new HW2 tests now

File: Code/sym.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# SE HW - 2
+
+class Sym:
+    def __init__(self, column_name, column_position):
+        self.column_name = column_name
```
Comment (kaushikjadhav01) [inline Code/sym.py:5]: Please use the same variable and function names given in lua file for consistency, like c, s or n, at, name, _has. Using different names will cause a lot of confusion in upcoming homeworks as code gets piled up

File: test/test_sample.py [hunk 3]
```diff
@@ -4,6 +4,9 @@
 
 import sys
 from Code.main import increment, decrement, square, divide
+from Code.sym import Sym
+
```
Comment (kaushikjadhav01) [inline test/test_sample.py:8]: Please change file name to test_eg.py and name of tests to for example: test_eg_sym()

Also, are we allowed to use pytest? Bcoz in lua he has written his own test suite and printed failures and everything manually using print statements. If we use pytest, our final output on terminal might deviate slightly from the lua output. I think for now we can submit HW2 with pytest and ask in discord on tuesday. If they tell us to make it from scratch, we can resubmit HW2 later

**Review by kaushikjadhav01 — CHANGES_REQUESTED** [review]

File: Code/sym.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        self.column_position = column_position
```
Comment (kaushikjadhav01) [inline Code/sym.py:6]: Give default values as c or 0 and s or '', just to keep the format consistent with lua. You can refer to my PR: https://github.com/Surya-06/SE-Proj22-Team11/pull/21/files

File: Code/sym.py [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        self.data = list()
```
Comment (kaushikjadhav01) [inline Code/sym.py:7]: Here, I assume your data, is 'n' in lua code. n is a number, not list. It represents the count of the total no. of items encountered so far.

Comment (Surya-06) [reply Code/sym.py:7]: As discussed, this will be taken up on a different thread.

File: Code/sym.py (hunk 4 above)
Comment (Surya-06) [reply Code/sym.py:6]: As discussed, this will be taken up on a different thread.

File: test/test_sample.py (hunk 3 above)
Comment (Surya-06) [reply test/test_sample.py:8]: As discussed, this will be taken up on a different thread.

File: Code/main.py (hunk 1 above)
Comment (Surya-06) [reply Code/main.py:7]: As discussed, this will be taken up on a different thread.

File: Code/sym.py (hunk 2 above)
Comment (Surya-06) [reply Code/sym.py:5]: As discussed, this will be taken up on a different thread.

**Review by vemuriPradyumna — APPROVED** [review]

**Pull request merged**